src/udev/udev-event.c

```diff
@@ -994,8 +994,13 @@ void udev_event_execute_run(UdevEvent *event, usec_t timeout_usec) {
         ORDERED_HASHMAP_FOREACH_KEY(val, cmd, event->run_list, i) {
                 UdevBuiltinCommand builtin_cmd = PTR_TO_UDEV_BUILTIN_CMD(val);
                 char command[UTIL_PATH_SIZE];
+                ssize_t ret;
 
-                (void) udev_event_apply_format(event, cmd, command, sizeof(command), false);
+                ret = udev_event_apply_format(event, cmd, command, sizeof(command), false);
+                if (ret < 0) {
+                        log_device_debug_errno(event->dev, (int) ret, "Failed to apply format to '%s', skipping to execute command: %m", cmd);
```

Comment:
Grammar seems a bit off here. Maybe `"Failed to format \"%s\", skipping command: %m"` ?